01-bash-scripting/02_connectivity_list_of_websites_2.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ $# -eq 0 ]; then
+    echo "Please provide at least 1 argument!"
+    exit 1
+fi
+
+for website in "$@"; do
+    if curl --output /dev/null --silent --head --fail "$website"; then
```

Comment:
There is no need for `--silent` and `--output /dev/null` together. `--silent` is enough for this use case.